app/Filament/Resources/GameResource/Pages/SimilarGames.php

```diff
@@ -170,7 +170,6 @@ public function table(Table $table): Table
                         Notification::make()
                             ->success()
                             ->title('Success')
-                            ->body(count($gameIds) > 1 ? 'Added similar games.' : 'Added similar game.')
```

Comment:
It's actually possible for nothing to be added on a successful submit, so I've removed this body text.